contracts/erc20-gateway/ERC20Cogateway.sol

```diff
@@ -203,4 +218,103 @@ contract ERC20Cogateway is
             messageHash_
         );
     }
+
+    /**
+     * @notice Confirm deposit in order to mint tokens.
+     *
+     * @param _amount ERC20 token deposit amount in atto.
+     * @param _beneficiary Address of beneficiary on target metachain.
+     * @param _feeGasPrice Gas price at which fee will be calculated.
+     * @param _feeGasLimit Gas limit at which fee will be capped.
+     * @param _valueToken ERC20 token address.
+     * @param _depositor Address of depositor on the origin chain.
+     * @param _blockNumber Block number of origin chain against which storage
+                           proof is generated.
+     * @param _rlpParentNodes Storage merkle proof to verify message declaration
```

Comment:
Please add `\pre` and `\post` documentation.

Reply:
Added.